src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -333,14 +339,13 @@ export class ShellIntegrationAddon extends Disposable implements IShellIntegrati
 				return true;
 			}
 			case VSCodeOscPt.CommandLine: {
-				let commandLine: string;
-				if (args.length >= 1 || args.length <= 2) {
-					commandLine = deserializeMessage(args[0]);
-				} else {
-					commandLine = '';
+				const commandLine = deserializeMessage(command);
```

Comment:
`command` here is `E` (the 633 sequence command), `arg[0]` is the command line that was run, `arg[1]` is the nonce.

https://github.com/microsoft/vscode/blob/3853550048a1dc738101b1579e0ee3dc8f3b45b3/src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts#L118